Exoft.Gamification.Api.Common/Models/User/UpdateFullUserModel.cs

```diff
@@ -1,7 +1,9 @@
-﻿namespace Exoft.Gamification.Api.Common.Models.User
+﻿using System.Collections.Generic;
+
+namespace Exoft.Gamification.Api.Common.Models.User
 {
     public class UpdateFullUserModel : UpdateUserModel
     {
-        public string Role { get; set; }
+        public List<string> Roles { get; set; }
```

Comment:
maybe we need to auto-initialize empty list here? this way we can avoid null checks below in code execution